docs/developer/audit-logs.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: audit-logs
+title: Audit logs
+sidebar_label: Audit logs
+sidebar_position: 6
+---
+
+# Audit logs
+
+Logto's audit log allows you to easily monitor user activity and events. It provides a strong foundation for various user management and health check business scenarios.
+
+## View all logs
+
+Navigate to **Console > Audit logs**. Logto captures and organizes authentication events into a table. It keeps track of the event name, user, application, and timestamp. You can narrow down the results by filtering based on the event name and application name. Clicking on a specific event will provide additional details.
```

Comment:
```suggestion
Navigate to <CloudLink to="/audit-logs">Console > Audit logs</CloudLink>. Logto captures and organizes authentication events into a table. It keeps track of the event name, user, application, and timestamp. You can narrow down the results by filtering based on the event name and application name. Clicking on a specific event will provide additional details.
```

Reply:
Good, updated all console navigation in the "developers" part